staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -107,18 +107,35 @@ type ValidatingAdmissionPolicySpec struct {
 	MatchConstraints *MatchResources `json:"matchConstraints,omitempty" protobuf:"bytes,2,rep,name=matchConstraints"`
 
 	// Validations contain CEL expressions which is used to apply the validation.
-	// A minimum of one validation is required for a policy definition.
+	// Validations and AuditAnnotations may not both be empty; a minimum of one Validations or AuditAnnotations is
+	// required.
 	// +listType=atomic
-	// Required.
-	Validations []Validation `json:"validations" protobuf:"bytes,3,rep,name=validations"`
+	// +optional
+	Validations []Validation `json:"validations,omitempty" protobuf:"bytes,3,rep,name=validations"`
 
-	// FailurePolicy defines how to handle failures for the admission policy.
-	// Failures can occur from invalid or mis-configured policy definitions or bindings.
+	// FailurePolicy defines how to handle failures for the admission policy. Failures can
+	// occur from CEL expression parse errors, type check errors, runtime errors and invalid
+	// or mis-configured policy definitions or bindings.
+	//
 	// A policy is invalid if spec.paramKind refers to a non-existent Kind.
 	// A binding is invalid if spec.paramRef.name refers to a non-existent resource.
+	//
+	// FailurePolicy does not define how Validations that evaluate to false are handled.
+	//
+	// When FailurePolicy is set to Fail, ValidatingAdmissionPolicyBinding ValidationActions
+	// define how failures are enforced.
+	//
 	// Allowed values are Ignore or Fail. Defaults to Fail.
 	// +optional
 	FailurePolicy *FailurePolicyType `json:"failurePolicy,omitempty" protobuf:"bytes,4,opt,name=failurePolicy,casttype=FailurePolicyType"`
+
+	// AuditAnnotations contains CEL expressions which are used to produce audit
+	// annotations for the audit event of the admission request.
+	// Validations and AuditAnnotations may not both be empty; a minimum of one Validations or AuditAnnotations is
+	// required.
+	// +listType=atomic
+	// +optional
+	AuditAnnotations []AuditAnnotation `json:"auditAnnotations,omitempty" protobuf:"bytes,5,rep,name=auditAnnotations"`
```

Comment:
consider how many items we should allow in this list.

Reply:
I've set this to 20 per ValidatingAdmissionPolicy. Surely this is higher than anyone will ever need :)